wandb/sdk/wandb_settings.py

```diff
@@ -306,6 +306,8 @@ class Settings(BaseModel, validate_assignment=True):
     x_service_transport: str | None = None
     x_service_wait: float = 30.0
     x_show_operation_stats: bool = True
+    # Whether to skip saving the run events to the transaction log.
```

Comment:
Please copy the full blob from above here

Reply:
it's kinda annoying to do it for all the settings... we need to find something less tedious